nerdlets/create-flow/blank.js

```diff
@@ -1,45 +1,23 @@
 import React, { useEffect, useMemo, useState } from 'react';
 import PropTypes from 'prop-types';
 
-import { Button, HeadingText, TextField } from 'nr1';
+import { Button, HeadingText, InlineMessage, TextField } from 'nr1';
 
 import { Select } from '../../src/components';
 import { REFRESH_INTERVALS } from '../../src/constants';
 
-const BlankFlow = ({ accountId, accounts = [], onCreate, onCancel }) => {
+const BlankFlow = ({ accountId, accountName, onCreate, onCancel }) => {
   const [flowName, setFlowName] = useState('');
   const [selectedAccountId, setSelectedAccountId] = useState();
+  const [selectedAccountName, setSelectedAccountName] = useState();
   const [selectedRefreshInterval, setSelectedRefreshInterval] = useState(
     REFRESH_INTERVALS[0]
   );
 
   useEffect(() => {
     if (!selectedAccountId) setSelectedAccountId(accountId);
-  }, [accountId, selectedAccountId]);
-
-  const accountsSelect = useMemo(
-    () =>
-      accounts.reduce(
-        (acc, { id, name }) => {
-          if (selectedAccountId === id) acc.selected = { id, name };
-
-          acc.items.push({
-            id,
-            selected: selectedAccountId === id,
-            option: (
-              <div className="account-picker-option">
-                <span>{name}</span>
-                <span>{id}</span>
-              </div>
-            ),
-          });
-
-          return acc;
-        },
-        { items: [], selected: null }
-      ),
-    [accounts, selectedAccountId]
-  );
+    if (!selectedAccountName) setSelectedAccountName(accountName);
+  }, [accountId, selectedAccountId, selectedAccountName]);
```

Comment:
I should have mentioned this in my previous review, but you can remove this whole block. Just use the `accountName` that's being passed to the component as a prop.